code/game/machinery/crusher_piston.dm

```diff
@@ -87,9 +87,15 @@
 	return ..()
 
 /obj/machinery/crusher_base/attackby(var/obj/item/O as obj, var/mob/user as mob)
+	if(status != "idle" && prob(40) && ishuman(user))
+		var/mob/living/carbon/human/M = user
+		M.apply_damage(45, BRUTE, "[prob(50) ? "r_hand" : "l_hand"]")
```

Comment:
I think you should use the active hand of the user here and not just a random hand.